batch/scripts/load_vector_csv_data.sh

```diff
@@ -6,22 +6,51 @@ set -e
 # -d | --dataset
 # -v | --version
 # -s | --source
-# -l | --source_layer
-# -f | --local_file
-# -X | --zipped
+
+# optional arguments
+# -g | --geometry_name (get_arguments.sh specifies default)
 
 ME=$(basename "$0")
 . get_arguments.sh "$@"
 
+set -u
```

Comment:
So presumably this will happen now whether it's table creation or an append? Have you tested this on a very large table to see if it runs into memory issues? (IIRC the temp tables are just stored in-memory) Since we break up the load data calls into smaller jobs, I think this shouldn't be an issue?

Reply:
As mentioned in Slack, this approach works with WDPA once upgrading the dev writer to the same instance class as staging/production.